### DISCUSSION TITLE
Performance with increasing the number of output sections/plugins in a multi-tenant Kubernetes environment

### DISCUSSION THREAD

**Discussion by adiforluls** [fluent/fluent-bit#7236] [Q&A]
I have a use case where I'm deploying a FluentBit daemonset using Fluent Operator in a multi-tenant/namespace Kubernetes environment. Each tenant will have a dedicated output section ([OUTPUT]) for their namespace to write logs to a destination. I'd like to know if there can be performance issues with increasing the number of output sections/plugins in FluentBit config.

**Comment by adiforluls** [comment]
@patrick-stephens or @edsiper would you know?

**Reply by patrick-stephens** [reply]
Duplicate of https://github.com/fluent/fluent-bit/discussions/7273?

**Reply by adiforluls** [reply]
Yes, I'll close the discussion
